cli/remote_download/remote_download.go

```diff
@@ -0,0 +1,128 @@
+package remote_download
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/buildbuddy-io/buildbuddy/cli/arg"
+	"github.com/buildbuddy-io/buildbuddy/cli/login"
+	"github.com/buildbuddy-io/buildbuddy/server/remote_cache/digest"
+	"github.com/buildbuddy-io/buildbuddy/server/util/flag"
+	"github.com/buildbuddy-io/buildbuddy/server/util/grpc_client"
+	"github.com/mattn/go-isatty"
+	"google.golang.org/protobuf/types/known/durationpb"
+
+	rapb "github.com/buildbuddy-io/buildbuddy/proto/remote_asset"
+	repb "github.com/buildbuddy-io/buildbuddy/proto/remote_execution"
+	rspb "github.com/buildbuddy-io/buildbuddy/proto/resource"
+	gstatus "google.golang.org/grpc/status"
+)
+
+var (
+	flags = flag.NewFlagSet("remote-download", flag.ContinueOnError)
+
+	target             = flags.String("target", login.DefaultApiTarget, "Remote gRPC target. Must support bytestream and remote asset APIs.")
```

Comment:
I would appreciate getting an API key flag here to target specific tenants.
Or if we can get the existing API key from .git/config by default, that would be nice as well.